packages/server-wallet/src/wallet/__test__/integration/join-channel.test.ts

```diff
@@ -40,25 +41,22 @@ describe('directly funded app', () => {
 
     await Channel.query(w.knex).insert(c2);
     const channelIds = [c1, c2].map(c => c.channelId);
-    const result = await w.joinChannels(channelIds);
-    expect(result).toMatchObject({
-      outbox: [
-        {
-          params: {
-            recipient: 'alice',
-            sender: 'bob',
-            data: {
-              signedStates: [
-                {...state1, turnNum: 1},
-                {...state2, turnNum: 1},
-              ],
-            },
-          },
-        },
-      ],
-      channelResults: [{channelId: c1.channelId}, {channelId: c2.channelId}],
+
+    const {outbox, channelResults} = await w.joinChannels(channelIds);
+
+    expect(getChannelResultFor(c1.channelId, channelResults)).toMatchObject({
+      channelId: c1.channelId,
+      turnNum: 1,
     });
 
+    expect(getChannelResultFor(c2.channelId, channelResults)).toMatchObject({
+      channelId: c2.channelId,
+      turnNum: 1,
+    });
+
+    expect(getSignedStateFor(c1.channelId, outbox)).toMatchObject({...state1, turnNum: 1});
+    expect(getSignedStateFor(c2.channelId, outbox)).toMatchObject({...state2, turnNum: 1});
```

Comment:
I'm not sure whether we should be testing what states go across the wire. It seems like it's a bit of an implementation detail.